rasa/core/training/dsl.py

```diff
@@ -197,6 +197,40 @@ def __init__(self, domain, interpreter, template_vars=None, use_e2e=False):
 
         return story_steps
 
+    def verify_stories_format(self, filename):
+        stories = self._remove_comments(filename)
+
+        for line in range(len(stories)):
+            stories[line] = stories[line].strip()
```

Comment:
why are we stripping? Also you can put this into the second loop so we don't have to loop twice.

Reply:
I added the strip method to check if the first char of the string is #, - , > or *. Without strip I could just be checking blank spaces.

Reply:
That makes sense. Just didn't know if stripping in this logic was going to affect anything later on.